examples/apps/cra-demo/src/App.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { KeyValueDataObject, KeyValueInstantiationFactory, IKeyValueDataObject } from "@fluid-experimental/data-objects";
+import { Fluid } from '@fluid-experimental/fluid-static';
+import { getContainerId } from './getContainerId';
+
+
+function useKVPair() {
```

Comment:
In general we'll need to fill this out with a lot more comments. I'm okay punting on this though

Reply:
i can add a little